package.json

```diff
@@ -1,29 +1,31 @@
 {
   "scripts": {
-    "start": "yarn run build && yarn run serve",
+    "start": "yarn run build && concurrently --raw 'yarn run watch-css' 'yarn run watch-js' 'yarn run serve'",
```

Comment:
What was that other option we were adding here? `--kill-all` or something? Could you add it here as well?